### PR TITLE
Simplify table note for max pods on a cluster

### PR DESCRIPTION
**Pull request by jab-rh**
Related to: https://cloud.redhat.com/blog/running-2500-pods-per-node-on-ocp-4.13

Version(s): 4.13+

Issue:

Link to docs preview: [OpenShift Container Platform tested cluster maximums for major releases](https://64061--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/planning-your-environment-according-to-object-maximums#cluster-maximums-major-releases_object-limits)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@sheriff-rh, do you know someone from Perf&Scale QE might be able to verify this? Thanks!

**Comment by sheriff-rh** [comment]
Hi @jab-rh , @mffiedler may be a good starting point.

**Comment by afcollins** [comment]
@jab-rh What is this waiting on? I got a question about the docs and was surprised to see it still mentions the (incorrect) OCS v4 in the note. I thought we closed on it her, but I see it was never merged.

**Comment by afcollins** [comment]
@mffiedler I understand from Jason this change needs QE sign-off. Could you please tag someone who might do that for us? Thanks!

**Comment by rpattath** [comment]
@qiliRedHat would you mind taking a look at this.

**Comment by qiliRedHat** [comment]
@afcollins I saw 'OCS v4'  had been removed, I'm confused about your comment below, let me know if I misunderstood what you meant. Would you mind double confirm the latest doc? I will give a qe sign-off with your confirmation.
> I got a question about the docs and was surprised to see it still mentions the (incorrect) OCS v4 in the note. I thought we closed on it her, but I see it was never merged.

File: modules/openshift-cluster-maximums-major-releases.adoc [hunk 1]
```diff
@@ -69,7 +69,7 @@ Red Hat does not provide direct guidance on sizing your {product-title} cluster.
...
-3. This was tested on a cluster with 31 servers: 3 control planes, 2 infrastructure nodes, and 26 worker nodes. The default `maxPods` is still 250. To get to 2,500 pods per node, the cluster must be created with `maxPods` set to `2500` using a custom kubelet config. If you need 2,500 user pods, you need a `hostPrefix` of `20` because there are 10-15 system pods already running on the node. The maximum number of pods with attached persistent volume claims (PVC) depends on the storage backend from where PVC are allocated. In our tests, only {rh-storage} v4 (OCS v4) was able to satisfy 2,500 of pods per node.
+3. This was tested on a cluster with 31 servers: 3 control planes, 2 infrastructure nodes, and 26 worker nodes. If you need 2,500 user pods, you need both a `hostPrefix` of `20`, which allocates a network large enough for each node to contain more than 2000 pods, and a custom kubelet config with `maxPods` set to `2500`. For more information, see link:https://cloud.redhat.com/blog/running-2500-pods-per-node-on-ocp-4.13[Running 2500 pods per node on OCP 4.13].
```
Comment (afcollins) [inline modules/openshift-cluster-maximums-major-releases.adoc:72]: Yes, this is correct iwtht he testing I did and looks good to me!

**Review by afcollins — APPROVED** [review]
LGTM, Thanks!

**Comment by afcollins** [comment]
Yes. I worked with Jason to make these changes, and I approve them.

To attempt to clarify about my comment around "OCS v4": This note was from the previous 500ppn blog and I did not use OCS with the 2500ppn testing. The only 2500ppn testing we've done is in the blog that would now be linked from the docs, so the docs and blog are now all in sync with what was actually completed.

> @afcollins I saw 'OCS v4' had been removed, I'm confused about your comment below, let me know if I misunderstood what you meant. Would you mind double confirm the latest doc? I will give a qe sign-off with your confirmation.

**Pull request merged**